src/Language/PureScript/Ide.hs

```diff
@@ -167,7 +167,7 @@ findAvailableExterns :: (Ide m, MonadError IdeError m) => m [P.ModuleName]
 findAvailableExterns = do
   oDir <- outputDirectory
   unlessM (liftIO (doesDirectoryExist oDir))
-    (throwError (GeneralError "Couldn't locate your output directory."))
+    (throwError (GeneralError $ "Couldn't locate your output directory: " <> (toS oDir) <> "."))
```

Comment:
What do you think of  `Couldn't locate your output directory at: ...`? Makes it a little clearer that we went searching for it.

Could we run the path through `System.FilePath.normalise` and use an explicit `T.pack` over `toS`? I also think we should leave off the trailing `.` since it'll be in the way when copy pasting the path.

Reply:
Makes sense.